sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1495,7 +1514,7 @@ def task_worker(self):
                             continue
 
                         # TODO: Use fine grained time when the CMIS memory map is available
-                        self.port_dict[lport]['cmis_expired'] = now + datetime.timedelta(seconds=self.CMIS_DEF_EXPIRED)
+                        self.port_dict[lport]['cmis_expired'] = now + datetime.timedelta(seconds=self.get_cmis_dp_init_duration(api))
                         self.port_dict[lport]['cmis_state'] = self.CMIS_STATE_DP_INIT
```

Comment:
We actually dont need this because at this stage DpInit is not triggered, it just we selected the application so we should check the DpInitPending bit at Page 11h Byte 235

Reply:
I can add code to check DpInitPending additionally but I am not sure if we can simply remove this line. If the module never sets the pending bits somehow, then it will get stuck in the CMIS_STATE_AP_CONF state.